taichi/program/kernel.cpp

```diff
@@ -185,15 +185,15 @@ void Kernel::LaunchContextBuilder::set_arg_float(int arg_id, float64 d) {
   }
 }
 
-void Kernel::LaunchContextBuilder::set_arg_int(int arg_id, int64 d) {
+void Kernel::LaunchContextBuilder::set_arg_int(int arg_id, uint64 d) {
   TI_ASSERT_INFO(!kernel_->args[arg_id].is_array,
                  "Assigning scalar value to external (numpy) array argument is "
                  "not allowed.");
 
   ActionRecorder::get_instance().record(
       "set_kernel_arg_int64",
```

Comment:
Shall we simply record "set_kernel_arg_integer" rather than "int64" (which isn't the case)?